asreview/webapp/src/ProjectComponents/ProjectModeSelect.js

```diff
@@ -42,8 +42,8 @@ export default function ProjectModeSelect(props) {
           </MenuItem>
           <MenuItem value={projectModes.EXPLORATION} divider>
             <SelectItem
-              primary="Exploration"
-              secondary="Explore or demonstrate ASReview LAB with a completely labeled dataset"
+              primary="Validation"
+              secondary="Validate labels provided by another screener or derived from a LLM, and explore benchmark datasets"
```

Comment:
Suggestion: a LLM or an AI? The latter might be more future proof.

Reply:
done! + updated screenshot